src/main/scala/com/typesafe/sbt/duplicates/DuplicatesFinderPlugin.scala

```diff
@@ -2,13 +2,15 @@ package com.typesafe.sbt.duplicates
 
 import java.io.File
 
-import sbt._
+import sbt.{Def, _}
```

Comment:
Given that all classes are imported, I don't think this is change is needed

Reply:
IntelliJ inserted this automatically, I'll delete that.